test/skeleton.js

```diff
@@ -70,23 +70,23 @@ describe('cp-organisations-service-controller', () => {
   // // VALIDATION
   // No need to validate them all, we just want to verify the bootloader (/lib/index)
   it('should validate entities acts with joi', (done) => {
-    seneca.act({ role: 'cd-organisations', entity: 'event', cmd: 'get', id: {} }, (err, app) => {
+    seneca.act({ role: 'cd-organisations', entity: 'org', cmd: 'get'}, (err, app) => {
       expect(err.code).to.be.equal('act_invalid_msg');
       done();
     });
   });
   //
   it('should validate entities acts while promised with joi', (done) => {
-    const app = seneca.export('cd-organisations/acts').event;
-    app.get({ id: {} })
+    const app = seneca.export('cd-organisations/acts').org;
+    app.get({ org: undefined})
```

Comment:
get takes an id : https://github.com/CoderDojo/cp-organisations-service/blob/master/lib/organisations/entities/org.js#L21

Reply:
you told me to put in org: undefined